specification/cognitiveservices/data-plane/TextTranslation/V3/TextTranslationV3.json

```diff
@@ -0,0 +1,930 @@
+{
```

Comment:
I noticed some places we call the service "Translator Text" some places we call it "Text Translation".
We should use same term.

If "Translator Text" is the official service name, in here, the path should be:
specification/cognitiveservices/data-plane/TranslatorText/v3.0/TranslatorText.json
```
TextTranslation => TranslatorText
V3 => v3.0
TextTranslationV3.json => TranslatorText.json
```